Wikipedia/Code/ExploreViewController.swift

```diff
@@ -19,6 +21,29 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
         isRefreshControlEnabled = true
         
         title = CommonStrings.exploreTabTitle
+
+        NotificationCenter.default.addObserver(self, selector: #selector(exploreFeedPreferencesDidSave(_:)), name: NSNotification.Name.WMFExploreFeedPreferencesDidSave, object: nil)
+    }
+
+    deinit {
+        NotificationCenter.default.removeObserver(self)
+    }
+
+    private func dismissCollapsedCards() {
```

Comment:
@nambatee In retrospect, this might be a bit expensive for the foreground, and it'd be good to group all the updates in one batch to the UI. Could this kick off an operation on the feed content controller?